internal/tiltfile/encoding/extension.go

```diff
@@ -8,18 +8,34 @@ func NewExtension() Extension {
 	return Extension{}
 }
 
+const (
+	readYAMLN         = "read_yaml"
```

Comment:
out of curiosity, what's the thinking behind moving these to N-suffixed constants?

Reply:
mostly that I wanted to reference the read_yaml_stream and decode_yaml_stream function names from the errors when read_yaml or decode_yaml get streams instead of docs